requirements-provider.txt

```diff
@@ -3,7 +3,7 @@ azure-storage-blob
 elasticsearch
 elasticsearch-dsl
 fiona
-#GDAL>=3.0.0
+GDAL>=3.0.0<=3.8.3
```

Comment:
Perhaps only <=3.8.3 ?

Reply:
Forgot to remove the previous pin, thanks